packages/server/src/tests/utilities/api/role.ts

```diff
@@ -21,20 +22,28 @@ export class RoleAPI extends TestAPI {
   }
 
   save = async (body: SaveRoleRequest, expectations?: Expectations) => {
+    // the tests should always be creating the "new" version of roles
+    if (body.version === undefined) {
```

Comment:
Should this not be part of the interface, then?

Reply:
true - I can detect if the role doesn't have a `_rev` set it